cvem/connectors/one/OpenNebula.py

```diff
@@ -148,10 +148,10 @@ def get_vm_list():
 			res = []
 			for vm in res_vm.VM:
 				host = HostInfo(int(vm.HISTORY_RECORDS.HISTORY[0].HID), vm.HISTORY_RECORDS.HISTORY[0].HOSTNAME)
-				new_vm = VirtualMachineInfo(int(vm.ID), host, vm.TEMPLATE.MEMORY * 1024, vm)
+				new_vm = VirtualMachineInfo(int(vm.ID), host, int(vm.TEMPLATE.MEMORY) * 1024, vm)
 				new_vm.user_id = vm.UID
 				if vm.USER_TEMPLATE.MEM_TOTAL:
-					new_vm.set_memory_values(int(vm.USER_TEMPLATE.MEM_TOTAL_REAL),
+					new_vm.set_memory_values(int(vm.REALMEMORY),
```

Comment:
The VM.real_memory value is used in the monitor to calculate the difference between the real_memory and the total_memory reported by the O.S. So this memory must be the value that in most of the cases is the same as allocated_memory and allways must be higher that total_memory (as stated in the comments (Total memory reported by the S.O. (less than real_memory) ) and not the current used memory.